src/client/src/OpenFin/apps/LimitChecker/LimitInput.tsx

```diff
@@ -1,16 +1,47 @@
 import styled from "styled-components"
 
 import { CurrencyPair } from "@/services/currencyPairs/types"
-import Input from "@/styleguide/components/Input"
 
 import { setLimit, useLimit } from "./state"
 
 const Container = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  background-color: ${({ theme }) => theme.core.darkBackground};
+  background-color: ${({ theme }) => theme.core.lightBackground};
   padding: 10px 10px 10px 10px;
```

Comment:
`padding: 10px` is the same as the expression.

Reply:
Changed